cli/lib/kontena/cli/stacks/service_generator.rb

```diff
@@ -64,6 +64,7 @@ def parse_data(options)
       data['build'] = parse_build_options(options) if options['build']
       data['health_check'] = parse_health_check(options)
       data['stop_grace_period'] = options['stop_grace_period'] if options['stop_grace_period']
+      data['read_only'] = options['read_only'] if options['read_only']
```

Comment:
Hmm... this only sends `{"read_only": true}` if it's set to true, but it never sends `{"read_only": false}`? How does that work if you `kontena stack upgrade` to change a service from `read_only: true` -> `read_only: false`?

Confirmed:
```
DEBUG [Request]: PUT http://localhost:9292/v1/stacks/development/readonly | Headers: {Accept: application/json, Content-Type: application/json, Authorization: Bearer}  | Body: 
{"name":"readonly","stack":"test/readonly","expose":null,"version":"0.0.1","source":"stack: test/readonly\nservices:\n  foo:\n    image: redis\n    read_only: false\n","registry":"file://","services":[{"instances":null,"image":"redis:latest","links":[],"external_links":[],"ports":[],"volumes":[],"volumes_from":[],"affinity":[],"stateful":false,"deploy_opts":{"wait_for_port":null,"min_health":null,"interval":null},"hooks":{},"health_check":{"port":null,"protocol":null,"uri":null,"timeout":null,"interval":null,"initial_delay":null},"log_driver":null,"log_opts":null,"name":"foo"}],"volumes":[],"variables":{}}
```

After changing stack file `read_only: false`, and `kontena stack upgrade`, the container is still readonly:
```
$ kontena service exec --shell readonly/foo 'echo foo > /foo'
/bin/sh: 1: cannot create /foo: Read-only file system
```